Makefile

```
@@ -0,0 +1,7 @@
+SHELL := /bin/bash
+
+cargo-run:
+	@cargo run --release
+	@cargo install --path .
+deploy-contract:
+	@. ./contracts/.env && . ./contracts/deploy.sh
```

don't inject .env here

```suggestion
	@. ./contracts/deploy.sh
```